core/extension/gdextension.cpp

```diff
@@ -675,7 +675,7 @@ GDExtensionInterfaceFunctionPtr GDExtension::get_interface_function(const String
 }
 
 Error GDExtension::open_library(const String &p_path, const Ref<GDExtensionLoader> &p_loader) {
-	ERR_FAIL_NULL_V_MSG(p_loader, FAILED, "Can't open GDExtension without a loader.");
+	ERR_FAIL_COND_V_MSG(p_loader.is_null(), FAILED, "Can't open GDExtension without a loader.");
```

Comment:
Not sure if it's related to your PR.
But, it's weird to have `is_null` and `is_valid` naming.
Depending on the context a `nullptr` value may be "valid".
So, why not `is_null` and `not_null`? `not_null` has the exact meaning of `!= null` while `is_valid` depends on a context.

Reply:
That's very unrelated, but that's a point for 5.0, but `is_valid` isn't context dependent, it means "the pointer is valid" it can't be non-null *and* valid, what would that mean? If it's a `Ref<Mesh>` for example it's either a valid pointer or not, it doesn't care about if it matches some abstract concept

Reply:
Sure! Thanks.